driver/src/main/java/eu/cloudnetservice/driver/service/ServiceConfigurationBase.java

```diff
@@ -196,19 +203,17 @@ protected ServiceConfigurationBase(
     }
 
     /**
-     * Adds the process parameters which should get appended to the command line. Process parameters are there to
+     * Modifies the process parameters which should get appended to the command line. Process parameters are there to
      * configure the application, for example setting an option like --online-mode=true.
      * <p>
-     * This method will override all previously added process parameters options. Furthermore, the given collection will
-     * be copied into this builder, meaning that changes to it will not reflect into the builder after the method call.
      * Duplicate parameters will get omitted by this method directly.
      *
-     * @param processParameters the process parameters to add to the configuration.
+     * @param modifier the process parameters modifier to apply to the collection.
```

Comment:
Would prefer that phrasing

```suggestion
     * @param modifier the modifier to be applied to the already added parameters of this builder.
```